src/lib.rs

```diff
@@ -934,6 +934,192 @@ impl Context {
         }
     }
 
+    /// Cause conditional gating of a policy based on locality.
+    ///
+    /// The TPM will ensure that the current policy can only complete in the specified
+    /// locality (extended) or any of the specified localities (non-extended).
+    pub fn policy_locality(
+        &mut self,
+        policy_session: ESYS_TR,
+        locality: TPMA_LOCALITY,
+    ) -> Result<()> {
+        let ret = unsafe {
+            Esys_PolicyLocality(
+                self.mut_context(),
+                policy_session,
+                self.sessions.0,
+                self.sessions.1,
+                self.sessions.2,
+                locality,
+            )
+        };
+        let ret = Error::from_tss_rc(ret);
+        if ret.is_success() {
+            Ok(())
+        } else {
+            Err(ret)
+        }
+    }
+
+    /// Cause conditional gating of a policy based on command code of authorized command.
+    ///
+    /// The TPM will ensure that the current policy can only be used to complete the command
+    /// indicated by code.
+    pub fn policy_command_code(&mut self, policy_session: ESYS_TR, code: TPM2_CC) -> Result<()> {
```

Comment:
Can `code` be any value from the `TPM2_CC` type values [here](https://github.com/parallaxsecond/rust-tss-esapi/blob/291180c19baa35936d87058fcfeace8aa0ea3879/src/constants.rs#L121)?

Reply:
Yes.

Reply:
Ok! My thought is that since the `TPM2_CC` and `TPMA_LOCALITY` types seem to be specific values amoung a range, it would be easy to implement an abstraction for those types as Rust enumerations with the `TryFrom` and `From` traits converting the enum variant from/to the specific values.
Just an idea, let's not do that in this PR!